spec/models/requests/mediated_page_spec.rb

```diff
@@ -44,13 +43,16 @@
       end.to raise_error(ActiveRecord::RecordInvalid, "Validation failed: I plan to visit on can't be blank")
     end
 
-    it 'allows requests to be submitted without a needed_date when not required' do
+    # TODO: CORY this seems to require a date now, should it?
+    xit 'allows requests to be submitted without a needed_date when not required' do
```

Comment:
@cbeer am I right in thinking that we no longer have mediated requests that DON'T require a needed_date, which I guess means I should remove this test.

Reply:
I think the `origin_location == 'PAGE-MP'` still exists?